Source/UIKit additions/SVGKFastImageView.m

```diff
@@ -6,6 +6,9 @@
 #import "SVGGradientElement.h" 
 #endif
 
+#define SetupLayerBackedView() \
+self.wantsLayer = YES;
```

Comment:
Is there a particular reason to make this a define, instead of just inlining it into the place where it's used?

Reply:
I can just remove these into inline code...Because I there are one `SetupLayerHostingView` macro in `SVGKLayeredView`.

Reply:
Done.